client/js/controllers/editor.js

```diff
@@ -589,7 +589,18 @@
                     return test;
                 }
 
+                function normalizeName(name) {
+                    if (name.charAt(0) === '.') {
+                        console.log(name.substr(1));
```

Comment:
I think this console.log can be removed as we are returning the required file/folder name.